docs/installation_linux.rst

```diff
@@ -36,6 +36,13 @@ Here is an example to install Python 3.11 on Ubuntu 22.04
    sudo apt update
    sudo apt install python3.11 python3.11-dev -y
 
+Here is an example to install Python 3.11 on Azure Linux (Mariner) 2.0 or 3.0
```

Comment:
Python is installed by the system depdencies step for Azure Linux, So it doesn't need to be installed here.

Reply:
This line is needed to specify, Azure Linux can skip this step, because it's installed by dependencies.